spec/jobs/delete_old_blackouts_job_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'spec_helper'
+
+describe DeleteOldBlackoutsJob, type: :job do
+  it 'deletes old blackouts' do
```

Comment:
Isn't this the same as the test you deleted for the other job? I think we can dispense with all of the "integration" style tests for jobs; what do you think?